src/reducers/books.js

```diff
@@ -0,0 +1,18 @@
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'CREATE_BOOK':
+      return [
+        ...state,
+        action.book,
+      ];
+    case 'REMOVE_BOOK':
+      return [
+        ...state.slice(0, state.indexOf[action.book]),
+        ...state.slice(state.indexOf[action.book] + 1),
+      ];
```

Comment:
- [ ] OPTIONAL - Consider using a `filter` function here. It's shorter and better.